docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -398,13 +398,49 @@ saved object ID of the action.
 
 |==============================================
 
+Optional fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|frequency |String a|Object containing action’s frequency:
+
+* `summary` (Boolean, required): Defines whether notification will be sent for summary of alerts or for each generated alert individually.
+
+* `notifyWhen` (String, required`): Defines how often alerts generate actions. Valid values are:
+
+** `onActiveAlert`: Actions run when the alert becomes active and at each check interval while the rule conditions are met.
+** `onThrottleInterval`: Actions run when the alert becomes active and at the interval specified in the throttle property while the rule conditions are met.
```

Comment:
@e40pud I removed the last part of the description because it seemed more related to what generates the alert, not how the alert is influenced by the `onThrottleInterval` field. Let me know if it's still necessary and I can re-add it in.

```suggestion
** `onThrottleInterval`: Actions run when the alert becomes active and at the interval specified in the throttle property.
```

Reply:
I copied description of these values from the alerting framework API docs https://www.elastic.co/guide/en/kibana/current/create-rule-api.html

Same as above, we might need to rephrase `alert becomes active` to `rule becomes active`

Reply:
@marshallmain would you mind reviewing this to make sure it's still technically accurate? TY!

```suggestion
** `onThrottleInterval`: Actions run on the specified throttle interval and summarize new alerts generated during that interval.
```

Reply:
That looks great!